src/averageAges.js

```diff
@@ -15,6 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let men = people.filter(man => man.sex === 'm');
+
+  if (century) {
+    men = men.filter(man => Math.ceil(man.died / 100) === century);
+  }
+
+  const sumOfAges = men
+    .map(man => man.died - man.born)
+    .reduce((sumOfAge, prevSum) => sumOfAge + prevSum);
```

Comment:
The variable name 'prevSum' in the reduce function is misleading. Consider renaming it to 'age' for better clarity.